src/kd/systimer/systimer.c

```diff
@@ -495,9 +514,10 @@ s32 sceSTimerSetTMCY(s32 timerId, s32 arg1)
     
     s32 val = TIMER_GET_COUNT(timer->regadr->nowData);
     s32 val2 = TIMER_GET_MODE(timer->regadr->nowData);
+    u32 prev_icnt = timer->prev_icnt;
```

Comment:
Do we need to store this in a temporary variable here, considering its

- only used once in the following code and
- simply storing a value directly here without futher modifications applied to it (different to `val` and `val2` above).


In case, we do want to keep the temporary variable: Can we add some spacing (an empty line) following this line? That way, we have the actual value setup and timer->... assignments visually separated.

Reply:
In code before my PR `timer->prev_icnt = val2;`  changes `prev_icnt` value before it's used again in `timer->totalcount` update. That didn't match FW module behavior. I guess we could swap assignment order here to avoid temp variable but the FW module assigns in this order.

Empty line I can add of course.